builtInFunctions/migrateUserName_test.go

```diff
@@ -0,0 +1,93 @@
+package builtInFunctions
+
+import (
+	vmcommon "github.com/multiversx/mx-chain-vm-common-go"
+	"github.com/multiversx/mx-chain-vm-common-go/mock"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"testing"
+)
+
+func TestNewMigrateUserNameFunc(t *testing.T) {
+	m, err := NewMigrateUserNameFunc(0, nil, nil)
```

Comment:
Would've refactored it like this in case someone changes nil check pointer logic inside constructor, this test would pass anyway.

```suggestion
	m, err := NewMigrateUserNameFunc(0, nil, &mock.EnableEpochsHandlerStub{})
```

Reply:
left as it is.